mkdocs.yml

```diff
@@ -147,21 +147,21 @@ nav:
     - Guides:
           - guides/index.md
           - Development:
-            - Hosting: guides/host.md
-            - Profiles & Settings: guides/settings.md
-            - Logging: guides/logs.md
-            - Testing: guides/testing.md
-            - Runtime Context: guides/runtime-context.md
-            - Variables: guides/variables.md
-            - Running flows with Docker: guides/docker.md
+                - Hosting: guides/host.md
+                - Profiles & Settings: guides/settings.md
+                - Logging: guides/logs.md
+                - Testing: guides/testing.md
+                - Runtime Context: guides/runtime-context.md
+                - Variables: guides/variables.md
           - Execution:
-            - Webhooks: guides/webhooks.md
-            - State Change Hooks: guides/state-change-hooks.md
-            - Dask & Ray: guides/dask-ray-task-runners.md
-            - Moving data: guides/moving-data.md
+                - Webhooks: guides/webhooks.md
+                - State Change Hooks: guides/state-change-hooks.md
+                - Dask & Ray: guides/dask-ray-task-runners.md
+                - Moving data: guides/moving-data.md
           - Workers and agents:
                 - Upgrade from Agents to Workers: guides/upgrade-guide-agents-to-workers.md
                 - Storage: guides/deployment/storage-guide.md
+                - Docker: guides/docker.md
```

Comment:
If I remember correctly we decided on moving the Docker guide to the `Development` section because the updated Docker guide doesn't reference workers or agents anymore.

Reply:
Ah. Makes sense. It got moved in the nav but not the index. Will move them both that direction and make sure there's a redirect that works for that. Thank you!

Reply:
Yeah, I prefer that to `Development`

Reply:
Cool. Adding to execution.